tests-behat/grid.feature

```diff
@@ -37,6 +37,20 @@ Feature: Grid
     Then No toast should be displayed
     Then PATCH MINK the url should match "~_unit-test/grid-rowclick.php#test~"
 
+  Scenario: master checkbox
+    Given I am on "_unit-test/grid-rowclick.php"
+    When I press button "Show Selection"
+    Then Toast display should contain text "Selected: #"
+    When I click using selector "//tr[1]//div.ui.child.checkbox"
+    Then I press button "Show Selection"
+    Then Toast display should contain text "Selected: 1#"
+    When I click using selector "//tr//div.ui.master.checkbox"
+    Then I press button "Show Selection"
+    Then Toast display should contain text "Selected: 1,2,3,4,5#"
+    When I click using selector "//tr//div.ui.master.checkbox"
+    Then I press button "Show Selection"
+    Then Toast display should contain text "Selected: #"
+
```

Comment:
#1920 adds support for non-UA (atk4/data User Action) bulk support

before we finish this PR, another PR should be made to add bulk support for "regular UA for multiple records" - https://github.com/atk4/data/blob/4.0.0/src/Model/UserAction.php#L32

Reply:
I have removed `APPLIES_TO_MULTIPLE_RECORDS` for now in favor of `APPLIES_TO_ALL_RECORDS` until implemented fully.